ui/apps/platform/cypress/integration/clusters/delegatedScanning.test.js

```diff
@@ -87,5 +97,20 @@ describe('Delegated Image Scanning', () => {
                 cy.get('h1:contains("Delegated Image Scanning")').should('not.exist');
             });
         });
+
+        it(`should not have a link on the Clusters page`, () => {
```

Comment:
Nit: it might be useful to specify that the page does/does not have a link here due to permission checks, versus other reasons.

e.g. `'should not have a link on the Clusters page when the user has insufficient permissions'`